geo-types/src/geometry.rs

```diff
@@ -69,6 +85,9 @@ impl<T: CoordinateType> Geometry<T> {
     /// let p2: Point<f32> = g.into_point().unwrap();
     /// assert_eq!(p2, Point::new(0., 0.,));
     /// ```
+    #[deprecated(
+        note = "Will be removed in an upcoming version. Switch to std::convert::TryFrom<Point>"
```

Comment:
shouldn't this, and all the following doc comments, be `std::convert::TryInto<...>`?

```suggestion
        note = "Will be removed in an upcoming version. Switch to std::convert::TryInto<Point>"
```

Reply:
Oops. Yep!